kolibri/plugins/coach/assets/src/views/reports/ReportsQuizLearnerListPage.vue

```diff
@@ -125,16 +123,35 @@
     beforeMount() {
       this.filter = this.filterOptions[0];
     },
+    methods: {
+      completedQuestionsCountLabel(answered, total) {
+        if (answered === total) {
+          return this.$tr('allQuestionsAnswered');
+        } else {
+          return this.$tr('questionsCompletedRatioLabel', { count: answered || 0, total: total });
+        }
+      },
+    },
     $trs: {
       allQuizzes: 'All quizzes',
       activeQuizzes: 'Active quizzes',
       inactiveQuizzes: 'Inactive quizzes',
+      allQuestionsAnswered: 'All questions answered',
       questionsCompletedRatioLabel:
-        '{count, number, integer} of {total, number, integer} {count, plural, other {answered}}',
+        '{count, number, integer} of {total, number, integer} questions {count, plural, other {answered}}',
     },
   };
 
 </script>
 
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+
+  .small-answered-count {
+    display: block;
+    margin-left: 1.75rem;
+    font-size: small;
+    color: gray;
```

Comment:
We should avoid hard-coding any colors - they should all come from the theme.

In this case, the proper color is probably `$themeTokens.annotation`